server/views/pages/index.njk

```diff
@@ -32,6 +32,10 @@
       <li><a class="govuk-link--no-visited-state" href="/product-dependencies">Product Dependencies</a></li>
       <li><a class="govuk-link--no-visited-state" href="/drift-radiator">Deployment Drift Radiator</a></li>
     </ul>
+    <h2>Requst Forms</h2>
+    <ul>
+      <li><a class="govuk-link--no-visited-state" href="/new-github-repo-request-form">Github repository request form (Replacing hmpps-projects-bootstrap projects.json)</a></li>
```

Comment:
Again this needs to match your route....so `/form/.....`

Reply:
Updated